packages/deprecation-crawler/src/lib/models.ts

```diff
@@ -73,7 +83,7 @@ export interface RawDeprecation {
 }
 
 export interface CrawledRelease {
-  gitTag: string;
+  tag: string;
```

Comment:
@BioPhoton I removed the `branch` property here as it wasn't used.
I was confused by it 😅